typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -103,4 +103,7 @@ export enum WarpRouteIds {
   // HYPER routes
   HYPER = 'HYPER/arbitrum-base-bsc-ethereum-optimism',
   stHYPER = 'stHYPER/bsc-ethereum',
+
+  // Testnet Routes
+  RebalancerTestnet = 'USDC/arbitrumsepolia-modetestnet-optimismsepolia-sepolia',
```

Comment:
weird that testnet routes go in the mainnet3 warpIds list, but idk if we have any alternatives